frame/system/src/lib.rs

```diff
@@ -180,11 +181,20 @@ impl WeightInfo for () {
 	fn suicide(_n: u32, ) -> Weight { 1_000_000_000 }
 }
 
+/// An object to track the currently used extrinsic weight in a block.
+pub type ExtrinsicsWeight = PerDispatchClass<Weight>;
```

Comment:
Naming proposal: If someone like me with poor eye sight does not read the `s` following `extrinsic`, (s)he thinks that this is the weight of _an extrinsic_, not the weights of the _extrinsics of a block_. Perhaps we can call this. Perhaps we can change this to `BlockWeight` and changed `type BlockWeights ` in ` Trait` to `BlockWeightConfig`. 

But I am just thinking out loud here, I think the naming is already better than the initial PR and it is fine.

Reply:
sounds good 👍